framework_tvb/tvb/adapters/exporters/export_manager.py

```diff
@@ -235,6 +239,18 @@ def export_project(self, project, optimize_size=False):
 
         return result_path
 
+    @staticmethod
+    def _get_op_with_errors():
+        """
+        Get the operation folders with error base name as list.
+        """
+        operations = dao.get_operations_in_group(None)
```

Comment:
This query might be inefficient. I think it is currently browsing ALL the projects for ops in error.
Wouldn't be better to have a dedicated query just for the current project ops in error ?